recipes/_install-linux.rb

```diff
@@ -78,5 +78,6 @@
     retries package_retries unless package_retries.nil?
     retry_delay package_retry_delay unless package_retry_delay.nil?
     action package_action # default is :install
+    allow_downgrade node['datadog']['agent_allow_downgrade']
```

Comment:
Is this valid with chef `>=12.7` (that's the lowest version supported by the cookbook)?

Reply:
Fixed in `3267da8`